zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementFacade.kt

```diff
@@ -180,6 +181,22 @@ class ObjectManagementFacade(
         }
     }
 
+    fun deleteObject(
+        objectName: String,
+        objectId: UUID
+    ): HttpStatus {
+        val accessObject = getAccessObject(objectName)
+
+        try {
+            logger.trace { "Deleting object $objectId" }
```

Comment:
Since this changes the state on a different system, want to log this as info. Maybe have the message be a bit more complete. Some thing like 

```
Deleting object '$objectId' of type '${accessObject.objectManagement.objecttypeId}' from Objecten API using plugin ${accessObject.objectManagement.objectenApiPluginConfigurationId}
```